src/kOS.Safe/Encapsulation/ScalarDoubleValue.cs

```diff
@@ -16,5 +16,10 @@ public ScalarDoubleValue(double value)
         {
             Value = value;
         }
+
+        public static implicit operator ScalarDoubleValue(double val)
+        {
+            return new ScalarDoubleValue(val);
+        }
```

Comment:
im not a fan of a ton of implicit converstions but for this we really are trying to say they are direct substitutes